configuration/default-applications.md

```diff
@@ -0,0 +1,125 @@
+
+This is probably a lot easier for users of xfce and Thunar; these instructions have been tested in Qubes R3.2 running i3 with. They have only been tested in Debian-based VMs and may need to be modified to work in Fedora-based VMs.
+
+Related documentation: [Using Disposable VMs](https://www.qubes-os.org/doc/dispvm/), [Tips and Tricks: Opening links in your preferred AppVM](https://www.qubes-os.org/doc/tips-and-tricks/#opening-links-in-your-preferred-appvm).
+
+
+## Setting default applications in Debian-based VMs.
+
+Some default applications can be set (system-wide) with the [Debian Alternatives System](https://wiki.debian.org/DebianAlternatives), noticeably the editor, terminal emulator and (sometimes) the web browser. You do this by running `sudo update-alternatives --config name` with `name` being e.g. `editor`, `x-terminal-alternatives`, (sometimes) `x-www-browser`, `vi`, etc. Other things you can set the default applications for are listed in the directory `/etc/alternatives/`. See `man update-alternatives` for more information about the command.
+
+For everything else, including URLs, you can set the default application according to [media (or MIME) types](https://en.wikipedia.org/wiki/Media_type).
+
+Run `mimetype filename` to get the MIME type of that file type. Example:
+
+	$ mimetype some_document.pdf
+	some_document.pdf: application/pdf
+
+Run `xdg-mime query default mime-type` to see which application opens this MIME type by default. Example:
+
+	$ xdg-mime query default application/pdf
+	evince.desktop
+
+To open a file in its default application run `xdg-open filename`.
+
+**Note:** If `xdg-mime` and/or `xdg-open` is not working as expected [you might need](https://askubuntu.com/questions/779717/xdg-open-stopped-working-since-16-04-upgrade) to install `qvfs-bin`. (This has happend in a debian-8 template updated to debian-unstable. It does not seem to happen in debian-minimal templates.)
+
+To set a new default application for a filetype run `xdg-mime default application.desktop mime-type`. `application.desktop` has to be a [Desktop entry](https://wiki.archlinux.org/index.php/Desktop_entries) in `/usr/source/applications/` or `~/.local/share/applications`. Example:
+
+	$ xdg-mime default zathura-pdf-poppler.desktop application/pdf
+
+`xdg-mime` confirms:
+
+	$ xdg-mime query default application/pdf
+	zathura-pdf-poppler.desktop
+
+This setting is usually effective immediately and is written to `~/.local/share/mimeapps.list` (whose first line is empty for some reason), i.e.:
+
+	
+	[Default Applications]
+	application/pdf=zathura-pdf-poppler.pdf
+
+See below for a more extensive `mimeapps.list` file.
+
+(If you want to apply this setting for every AppVM of a TemplateVM you can copy `mimeapps.list` to `/usr/share/applications/`.)
```

Comment:
I have changed the how-to so that the possibility of saving the files to `/usr/share/applications` is only mentioned in a note at the end.